XTS/jbossxts-jakarta/pom.xml

```diff
@@ -0,0 +1,270 @@
+                                  replace="-sources.jar"
+                                  global="true" />
+                <java classname="org.eclipse.transformer.jakarta.JakartaTransformer">
+                  <arg value="${source}"/>
+                  <arg value="${project.build.directory}/${project.build.finalName}-sources.jar"/>
+                  <arg value="-q"/>
+                  <arg value="-o"/>
+                  <arg value="-tr"/>
+                  <arg value="${project.basedir}/../../jakarta-rules/jakarta-renames.properties"/>
+                  <classpath>
+                    <pathelement path="${plugin_classpath}"/>
+                  </classpath>
+                </java>
+              </target>
+            </configuration>
+          </execution>
+          <execution>
+            <id>transform-javadoc</id>
+            <phase>package</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <property name="plugin_classpath" refid="maven.plugin.classpath"/>
+                <ac:propertyregex xmlns:ac="antlib:net.sf.antcontrib" property="source"
+                                  input="${org.jboss.narayana.xts:jbossxts:jar}"
+                                  regexp="\.jar$"
+                                  replace="-javadoc.jar"
+                                  global="true" />
+                <java classname="org.eclipse.transformer.jakarta.JakartaTransformer">
+                  <arg value="${source}"/>
+                  <arg value="${project.build.directory}/${project.build.finalName}-javadoc.jar"/>
+                  <arg value="-q"/>
+                  <arg value="-o"/>
+                  <arg value="-tr"/>
+                  <arg value="${project.basedir}/../../jakarta-rules/jakarta-renames.properties"/>
+                  <classpath>
+                    <pathelement path="${plugin_classpath}"/>
+                  </classpath>
+                </java>
+              </target>
+            </configuration>
+          </execution>
+        </executions>
+        <dependencies>
+          <dependency>
+            <groupId>org.eclipse.transformer</groupId>
+            <artifactId>org.eclipse.transformer.cli</artifactId>
+            <version>${version.org.eclipse.transformer}</version>
+          </dependency>
+          <dependency>
+            <groupId>ant-contrib</groupId>
+            <artifactId>ant-contrib</artifactId>
+            <version>${version.ant-contrib}</version>
+            <exclusions>
+              <exclusion>
+                <groupId>ant</groupId>
+                <artifactId>ant</artifactId>
+              </exclusion>
+            </exclusions>
+          </dependency>
+        </dependencies>
+      </plugin>
+      <plugin>
+        <inherited>false</inherited>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+      </plugin>
```

Comment:
This plugin can be removed as there is already in the `release` profile (thanks for moving it there :-) )